conf/app.src

```diff
@@ -1,6 +1,6 @@
-SOURCE_URL=https://github.com/RSS-Bridge/rss-bridge/archive/refs/tags/2022-06-14.tar.gz
-SOURCE_SUM=88daee9604adcb2193ee771c260bcf422a806c284a46c4f263d42cf323d42fe1
+SOURCE_URL=https://github.com/RSS-Bridge/rss-bridge/archive/refs/heads/master.zip
+SOURCE_SUM=c39b875e1b2c047662b06366f63a6888f6fba3c5146e2034433f06a7e40594d3
```

Comment:
```suggestion
SOURCE_URL=https://github.com/RSS-Bridge/rss-bridge/archive/307f5865c0bd12c32e7c5bdc0869dd0dcd403e40.zip
SOURCE_SUM=f539b420d153485a9495105bedf4197ae9c54ed5b391d4e70b3c403aa76bc96c
```
I would suggest to use a source archive for the specific commit number (I took the latest commit in the `master` branch at the moment). This way it should be deterministic between multiple installs and should download the same sources until the link will be changed again.

Also, this change would require a change of the package version  postfix at least `~ynh3..~ynh4` to notify users that package is changed.

Reply:
> Also, this change would require a change of the package version postfix at least ~ynh3..~ynh4 to notify users that package is changed.

If we change the commit version, the whole version name has to change.
I believe this PR will bring a newer version anyway.

Reply:
> If we change the commit version, the whole version name has to change.

Yes, then since Bridge developers are using the dates, it might be a date of the commit was used in update, e.g. `2023.03.21`.

Reply:
I would suggest as well to point to a specific commit archive, and change the package version each. An example in the ttrss package: https://github.com/YunoHost-Apps/ttrss_ynh/blob/master/manifest.toml#L43

Reply:
This issue is that it's a burden to maintain and test.
But if no new releases are going to be made, then we might have no choice.

Reply:
> This issue is that it's a burden to maintain and test. But if no new releases are going to be made, then we might have no choice.

How is the SearxNG managed? It seems to push updates every other day.